guides/common/modules/proc_monitoring-server.adoc

```diff
@@ -0,0 +1,13 @@
+[id="monitoring-{project-context}-server_{context}"]
+= Monitoring {ProjectServer}
+
+Audit records list the changes made by all users on {Project}.
+This information can be used for maintenance and troubleshooting.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Audits* to view the audit records.
+. To obtain a list of all the audit attributes, use the following command:
+[options="nowrap"]
```

Comment:
```suggestion
.Procedure
. In the {ProjectWebUI}, navigate to *Monitor* > *Audits* to view the audit records.

.CLI Procedure
. To obtain a list of all the audit attributes, use the following command:
[options="nowrap"]
```

Reply:
Hey, Akshay. I see, thanks for correcting me. In that case, it can stay as it is. I have no further comments so this can get merged unless @maximiliankolb wants to take a last look?